prac_06/guitars.py

```diff
@@ -0,0 +1,37 @@
+"""
+CP1404 Practical 06 - Guitar program using class and methods
+Time estimated: 50 minutes
+Time taken: 40 minutes
+"""
+from prac_06.guitar import Guitar
+
+
+def main():
+    """Program to store and display details of a user's guitars."""
+    guitars = []
+    print("My guitars!")
+    name = input("Name: ")
+
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        guitar_to_add = Guitar(name, year, cost)
+        guitars.append(guitar_to_add)
+        print(f"{guitar_to_add} added.")
+        name = input("Name: ")
+    # guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+    # guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+
+    if guitars:  # First check whether the list has contents
+        guitars.sort()
+        print("These are my guitars:")
+        for i, guitar in enumerate(guitars, 1):
+            vintage_string = ""
+            if guitar.is_vintage():
+                vintage_string = " (vintage)"
```

Comment:
Nice use of if statement for the vintage string, you could of also tried ternary operator to get it on 1 line
vintage_string = "(vintage)" if guitar.is_vintage() else ""